docs/pages/reference/cli/tsh.mdx

```diff
@@ -89,8 +89,9 @@ $ tsh clusters --quiet
 
 ## tsh config
 
-Generates OpenSSH configuration to use currently logged in teleport
-as a bastion host.
+Print OpenSSH configuration details to allow using a SSH
```

Comment:
```suggestion
Print OpenSSH configuration details to allow using an SSH
```
Since, sounded out, "SSH" would begin "ess": https://www.scribbr.com/commonly-confused-words/a-vs-an